tests/foreman/cli/test_http_proxy.py

```diff
@@ -97,12 +97,6 @@ def test_positive_create_update_delete(module_org, module_location, target_sat):
     indirect=True,
     ids=['auth_http_proxy', 'unauth_http_proxy'],
 )
-@pytest.mark.parametrize(
-    'module_target_sat_insights',
-    [True, False],
-    indirect=True,
-    ids=['hosted', 'local'],
-)
 def test_insights_client_registration_with_http_proxy(
```

Comment:
I think we're validating the same things twice here and in newly added test, so can't we remove this test now?